README.md

```diff
@@ -265,6 +266,28 @@ These variables must follow a specific naming convention, as listed in the [Boto
 
 A list of all locations checked for Amazon Web Services credentials can also be found [here](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials).
 
+### Neo4J
+
+Change the configuration using `%%graph_notebook_config` and modify the fields for `host`, `port`, `ssl`, and `neo4j` authentication. For a local Neo4j Desktop database, if querying via Bolt protocol, you can use the following command:
+
+```
+%%graph_notebook_config
+{
+  "host": "localhost",
+  "port": 7687,
+  "ssl": false,
+  "neo4j": {
```

Comment:
We need to add support here for specifying the database as well https://community.neo4j.com/t5/neo4j-graph-platform/how-to-specify-to-connect-to-a-particular-database-name-in/m-p/48122